src/main/kotlin/ecommerce/controller/api/AdminReportController.kt

```diff
@@ -0,0 +1,32 @@
+package ecommerce.controller.api
+
+import ecommerce.annotation.LoginMember
+import ecommerce.domain.Member
+import ecommerce.dto.report.MemberCartActivityDTO
+import ecommerce.dto.report.ProductCartCountDTO
+import ecommerce.service.ReportService
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/api/admin/reports")
+class AdminReportController(private val reportService: ReportService) {
```

Comment:
Why don't we keep naming consistent here as well by adding `Api` into the name, or dropping it from other controllers?